src/saltext/vmware/utils/common.py

```diff
@@ -604,6 +604,60 @@ def get_path(node, service_instance, path=""):
     return get_path(node, service_instance, path)
 
 
+def get_clusters(service_instance, datacenter_name=None, cluster_name=None):
+    """
+    Returns clusters in a vCenter.
+
+    service_instance
+        The Service Instance Object from which to obtain cluster.
+
+    datacenter_name
+        (Optional) Datacenter name to filter by.
+
+    cluster_name
+        (Optional) Exact cluster name to filter by. Requires datacenter_name.
+    """
+    if datacenter_name is None and cluster_name:
```

Comment:
**question** would it be more correct to just check for the truthiness, i.e. `if cluster_name and not datacenter_name:` - in other words, is `''` a valid cluster name?

Reply:
Fair point, done.